getting_started/publish_app.md

```diff
@@ -0,0 +1,108 @@
+# Publishing your First Project Firefly Application
+In this tutorial, we'll guide you through the following steps to give you an introduction on how to publish your first Firefly Application. 
+after you deployed your application, you need to go through below 3 steps to publish your headful app
+1. Submit for approval through [Adobe Developer Console](https://console.adobe.io/)
+2. Review and approve the app in [My Exchange](https://exchange.adobe.com/my-exchange.html)
+3. After approval, your app will be available at [Adobe Experience Cloud](https://experience.adobe.com/#/@adobeio/home)
```

Comment:
The link is incorrect (points to the @adobeio org).
Please use `https://experience.adobe.com` instead.